xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.Pc461-I313.xml

```diff
@@ -0,0 +1,137 @@
+                  </handDesc>
+
+                  <!--  *************************************  <decoDesc>  *********************************     -->
+                  <decoDesc>
+                     <decoNote>
+                        <p/>
+                     </decoNote>
+                  </decoDesc>
+
+               </physDesc>
+
+               <history><!--summary, origin, provenance go here-->
+
+                  <summary>
+                     <rs/>  <!-- content can be "fake" or blank -->
+                  </summary>
+
+                  <origin>
+                     <date notBefore="0301" notAfter="0500">fourth-fifth century CE</date>
+                     <placeName ref="asia.asia_minor">Turkey, Seleucia</placeName>
+                  </origin>
+
+
+                 <provenance type="transfer" subtype="given">
+                     <!-- subsequent locations if inscription was moved -->
+                     <p> Gift of the <orgName>Committee for the Excavation of Antioch</orgName> to <orgName>Princeton
+                        University</orgName> in <date when="2003">2003</date></p>
+                  </provenance>
+               </history>
+
+            </msDesc>
+
+            <listBibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#IGLS"/>
+                  <biblScope unit="item">1242.5</biblScope>
+               </bibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#Antioch_III"/>
+                  <biblScope unit="item">106, no. 222</biblScope>
+               </bibl>
+               <bibl>
+                  <ref type="external"
+                     target="https://artmuseum.princeton.edu/collections/objects/41920">Princeton
+                     University Art Museum</ref>
+               </bibl>
+            </listBibl>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <xi:include href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            <xi:fallback>
+               <p>Taxonomies for US Epigraphy controlled values</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc/>
+      <revisionDesc>
+         <change when="2010-09-29" who="CDS">Converted to September 2009 EpiDoc</change>
+         <change when="2011-05-13" who="EM">More tweaks for bib only inscriptions</change>
+         <change when="2020-02-05" who="SPapandonatou">added metadata</change>
+      </revisionDesc>
+   </teiHeader>
+   <!--facsimile element goes here--><text>
+      <body>
+         <div type="edition">
+
+         <ab>
+            <lb n="1"/><gap reason="lost" extent="unknown"/><gap reason="illegible" quantity="1" unit="character"/>ΠΡ<hi rend="superscript">E</hi><gap reason="lost" extent="unknown"/>
+            <lb n="2"/><gap reason="lost" extent="unknown"/><damage>Κ</damage>ΑΘ<hi rend="superscript">s</hi><unclear>Ε</unclear><gap reason="lost" extent="unknown"/>
```

Comment:
We can confer with @jbodel and Dr. Oliver about this; I think this is punctuation indicating an elision (καθ') but best to confirm. Otherwise this looks good to me.

Reply:
For reference, this is an image of the inscription in question: https://static.artmuseum.princeton.edu/mirador/?manifest=https://data.artmuseum.princeton.edu/iiif/objects/41920&canvas=https://data.artmuseum.princeton.edu/iiif/objects/41920/canvas/41920-canvas-6897.

Reply:
@spapando let me know that this was approved, so I'm resolving the issue and leaving it as is.